config/puma.rb

```diff
@@ -8,9 +8,14 @@
 min_threads_count = ENV.fetch("RAILS_MIN_THREADS") { max_threads_count }
 threads min_threads_count, max_threads_count
 
+# Specifies the `worker_timeout` threshold that Puma will use to wait before
```

Comment:
Haven't seen this on the other 6.1 bumps I don't think

Reply:
It is on a few other repos, e.g. https://github.com/alphagov/govuk-account-manager-prototype/blob/c3c628500493c0e08101dbda7879ef6af8241ee4/config/puma.rb and https://github.com/alphagov/locations-api/blob/af440521ec435afcce54dbdc7ece4c592c5494a8/config/puma.rb.